src/shared/sam/cli/samCliInvoker.ts

```diff
@@ -39,14 +40,18 @@ export function resolveSamCliProcessInvokerContext(
 export class DefaultSamCliProcessInvoker implements SamCliProcessInvoker {
     public constructor(private readonly context: SamCliProcessInvokerContext = resolveSamCliProcessInvokerContext()) {}
 
-    public async invoke(options?: SamCliProcessInvokeOptions): Promise<ChildProcessResult> {
+    public async invoke(
+        options?: SamCliProcessInvokeOptions,
+        channelLogger?: ChannelLogger
+    ): Promise<ChildProcessResult> {
         const invokeOptions = makeRequiredSamCliProcessInvokeOptions(options)
+        const logger = getLogger()
```

Comment:
nit: we don't actually have to hold the value of logger, it's fine either way though

Reply:
This is probably a good pattern to start following, because in the future I think `getLogger()` should accept a "topic" so that messages are prefixed with the topic:

    const logger = getLogger('myModule')
    logger.info('ok')
    // => INFO: myModule: ok